### PR TITLE
feat: [BREAKING_CHANGE] delete hackernews stuff

### PR DESCRIPTION
**Pull request by atian25**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

##### Description of change

移除 hackernews 相关内容.

- https://github.com/eggjs/egg/issues/218
- https://github.com/eggjs/egg-boilerplate-empty/pull/5

### CODE REVIEW

File: boilerplate/README.md [hunk 1]
```diff
@@ -4,6 +4,10 @@
 
 ## QuickStart
 
+<!-- add docs here for user -->
+
+see [egg docs][egg] for more detail.
```
Comment (popomore) [inline boilerplate/README.md:9]: 还能这样写？

**Review by popomore — APPROVED** [review]

File: boilerplate/README.md (hunk 1 above)
Comment (atian25) [reply boilerplate/README.md:9]: 嗯, 引用底部的标注

**Review by fengmk2 — APPROVED** [review]

**Comment by atian25** [comment]
我发个 major

**Comment by atian25** [comment]
2.0.0

**Pull request merged**